Update docs for `--light` mode (#3191)
input-output-hk/cardano-wallet: docs/user-guide/common-use-cases/how-to-start-wallet-server.md

}
```

## Light mode

> :warning: This mode is currently **under development**. Please note that some parts may be available only in `master` branch.

> You can start your cardano-wallet server also in **light mode**. As opposed to full-node mode, in light your wallet is not connected to locally running instance of `cardano-node`. Instead it relays on external source of blockchain data. This significantly improves synchronization speed of the wallet and also you don't need to spend time for synchronizing the node itself. The downside is that external source of data is obviously less trusted than the local one provided by your own `cardano-node` instance.

Reviewer: ```suggestion
You can start your cardano-wallet server also in **light mode**. As opposed to full-node mode, in light-mode your wallet is not connected to a locally running instance of `cardano-node`. Instead it relays on external source of blockchain data. This significantly improves synchronization speed of the wallet and also removes the need to spend time synchronizing the node itself. The downside is that external source of data is obviously less trusted than the local one provided by your own `cardano-node` instance.
```
